packages/vite/src/node/__tests__/plugins/define.spec.ts

```diff
@@ -20,21 +21,15 @@ describe('definePlugin', () => {
     const transform = await createDefinePluginTransform({
       __APP_VERSION__: JSON.stringify('1.0')
     })
-    expect(await transform('const version = __APP_VERSION__ ;')).toBe(
-      'const version = "1.0" ;'
-    )
     expect(await transform('const version = __APP_VERSION__;')).toBe(
-      'const version = "1.0";'
+      'const version = "1.0";\n'
```

Comment:
Esbuild transform tend to semi-format the entire code so this is changed here. That also means we can't do stable sourcemaps transform.